### PR TITLE
Kai infra for coolstore EAP deployment 

### PR DESCRIPTION
**Pull request by savitharaghunathan**
This role only installs coolstore EAP on the server
* removed docker stuff
* uses podman for postgres
* updated readme 
* updated dependencies

### CODE REVIEW

File: ec2/DeployEAP.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+### To create jboss wildfly instance
+
+* Make sure to set the role to `jboss-wildfly` in `my_vars.yml`
+
+Run `./setup_nodes.sh`
```
Comment (jwmatthews) [inline ec2/DeployEAP.md:5]: Do you want this line to stay here?

**Comment by jwmatthews** [comment]
Tested with downloading `https://github.com/keycloak/keycloak/releases/download/20.0.5/keycloak-20.0.5.zip` 

./create_infra.sh
./setup_coolstore_app.sh

Success :)

**Review by jwmatthews — APPROVED** [review]
Assume we want to address the keycloak side, or update docs with guidance for the download/offline mode.

Otherwise, verified works for me.

**Comment by savitharaghunathan** [comment]
@jwmatthews I added instructions for keycloak offline install here. I will merge this and work on the fix. The idea is to have something like a reproducible workflow while I am gone. let me know if that works for you.

File: ec2/roles/coolstore_deploy_eap/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+Requirements
+------------
+
+* Provision the server: Pre-req can be found in [readme](../README.md)
+Run `cd.. && ./create_infra.sh`
+
+* We will be using offline installation of Red Hat EAP for this role. Please [download](https://access.redhat.com/jbossnetwork/restricted/listSoftware.html?downloadType=distributions&product=appplatform&version=7.4) `jboss-eap-7.4.0.zip` from the RH Customer Portal and place it in `../ec2/jboss-eap-7.4.0.zip`. Make sure the file type is `zip`.
+
+* We will be using offline installation for keycloak. Download [keycloak-20.0.5.zip](https://github.com/keycloak/keycloak/releases/download/20.0.5/keycloak-20.0.5.zip) and place it under ../ec2
+File relative path eg: ec2/keycloak-20.0.5.zip
```
Comment (jwmatthews) [inline ec2/roles/coolstore_deploy_eap/README.md:15]: LGTM

**Review by jwmatthews — APPROVED** [review]
LGTM, please merge when you are ready

**Pull request merged**